### ISSUE TITLE
 Accessibility issue for Dropdown component

### ISSUE THREAD

**Issue by jz2cao** [carbon-design-system/carbon#6853]
## Title line template: Accessibility issue for Dropdown component

-->

## Environment 
All environments

> Operating system
MAC

> Browser
Chrome

> Automated testing tool and ruleset
IBM Accessibility
Equal Access Accessibility Checker

## Detailed description

> What version of the Carbon Design System are you using?
    "carbon-components": "10.16.0",
    "carbon-components-react": "7.16.0",

> What did you happened?
For `Dropdown` component:
![image](https://user-images.githubusercontent.com/15077843/93393128-ee212880-f83f-11ea-8349-1b1b0e779bb8.png)

> What did you expect to happen?
We should not see those `WAI-ARIA` violation.

> Please create a reduced test case in CodeSandbox
We are seeing similar issue in the CodeSandbox when doing similar scan against the sandbox sample

> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

**Comment by xiaolixu** [comment]
I encounted the same accessbility issue just with the code 
`<Dropdown direction="top" />`

**Comment by jz2cao** [comment]
@rjhenriquez 
any update on the issue?

**Comment by rjhenriquez** [comment]
@jz2cao I wasn't tracking this issue, but I'll take a look at it

**Comment by dakahn** [comment]
@jz2cao hey there. If you can try and provide an example in Codesandbox for us to test. It can greatly speed this process along; that said, does your Dropdown have a label? According to the error Downshift (the library we use for our Dropdown under the hood) has generated an id for the `aria-labelledby` attribute on the input, but that id isn't assigned to a corresponding label. These form elements require visible and programmatic labels to be compliant 👍

**Comment by xiaolixu** [comment]
When the issue is seen, there is no label for the Dropdown component. In some design requirement, the label of the Dropdown should be hiddened.

**Comment by tw15egan** [comment]
@xiaolixu I believe visible labels are an accessibility requirement and hiding them will result in a violation

**Comment by tw15egan** [comment]
Closing this issue as it is an AVT violation to _NOT_ provide a visible label. This issue is occurring because no label is being provided.

**Comment by jz2cao** [comment]
@tw15egan 
Right we did not provide a label for the dropdown field and that is because the design requires us not to provide a label for that field. In that case, is there a work around for us to resolve the violation?

e.g. We didn't provide the label because we don't want the label to show up in the UI. Is there other way we can provide the label information so that it WILL NOT show up in the UI and WILL NOT violate the accessibility rules?
